lib/index.js

```diff
@@ -186,8 +186,8 @@ NwBuilder.prototype.resolveLatestVersion = function () {
 
 NwBuilder.prototype.checkVersion = function () {
     var version = this.options.version,
-        flavor = semver.satisfies(version, '<0.12.3') ? 'sdk' : this.options.flavor,
         self = this;
+    var flavor = semver.satisfies(version, '<0.12.3') ? 'sdk' : this.options.flavor;
```

Comment:
OOPS, this is not the right position. 

Should move to the line above `var getVersionFromManifest = function(){`